resources/release-notes.md

```diff
@@ -10,6 +10,34 @@ description: |
 body="The version numbers used in headers on this page refers to the version of
 this very documentation, not to a version of any APIs described by it." %}
 
+
+## 11 Nov 2020
+
+### Version 2.0.0
+
+:tada: Launched new developer portal theme: :tada:
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added Transaction on File section.
```

Comment:
Can we get a link to transaction on file here?

Reply:
Oops! Sure!